common4j/src/main/com/microsoft/identity/common/java/controllers/BaseController.java

```diff
@@ -358,12 +358,10 @@ private void setBuilderProperties(@SuppressWarnings(WarningType.rawtype_warning)
 
         if (builder instanceof MicrosoftStsAuthorizationRequest.Builder) {
             final MicrosoftStsAuthorizationRequest.Builder msBuilder = (MicrosoftStsAuthorizationRequest.Builder) builder;
-            msBuilder.setLoginHint(
-                    interactiveTokenCommandParameters.getLoginHint()
-            ).setPrompt(
-                    interactiveTokenCommandParameters.getPrompt().toString()
-            );
-
+            msBuilder
+                    .setLoginHint(interactiveTokenCommandParameters.getLoginHint())
+                    .setPrompt(interactiveTokenCommandParameters.getPrompt().toString())
+                    .setPreferredAuthMethod(interactiveTokenCommandParameters.getPreferredAuthMethod());
```

Comment:
Is preferredAuthMethod nullable?

Reply:
Sorry, missed that part of the description. So if we don't pass a preferred method we just use our standard behavior

Reply:
correct